### PR TITLE
README and file structure updates for whereami app

### PR DESCRIPTION
**Pull request by mark-church**
- Updated README
  - Separated into 3 sections - simple deployment, full deployment, and backend service call
  - Added descriptions to YAML examples to highlight areas of the config
- Moved the entire folder from gke-whereami to whereami. Reason for this is that the app is not actually GKE specific so I wanted to give it a little bit more of a generic name. 
- Created a k8s-frontend-overlay-example folder so we could keep the "full deployment" config a little cleaner and completely separate their configs

### CODE REVIEW

File: whereami/k8s-frontend-overlay-example/cm-flag.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: whereami-configmap
+data:
+  BACKEND_ENABLED: "True" 
+  BACKEND_SERVICE: "whereami-backend" # substitute with corresponding service name - this example assumes both services are in the same namespace  
+  METADATA:        "frontend"
```
Comment (askmeegs) [inline whereami/k8s-frontend-overlay-example/cm-flag.yaml:8]: @mark-church I'm seeing some extra whitespace in line 8

**Comment by mark-church** [comment]
@theemadnes let me know if you are okay with these changes and I will submit

File: whereami/k8s-frontend-overlay-example/cm-flag.yaml (hunk 1 above)
Comment (mark-church) [reply whereami/k8s-frontend-overlay-example/cm-flag.yaml]: fixed the whitespace

**Comment by mark-church** [comment]
@askmeegs btw I don't have write access to the repo so I can't merge myself

**Pull request merged**